jsk_fetch_robot/jsk_fetch_startup/config/test_device_topics.yaml

```diff
@@ -0,0 +1,29 @@
+topic_0: /audio
+timeout_0: 10
+
+topic_1: /base_scan
+timeout_1: 10
+
+topic_2: /battery_state
+timeout_2: 10
+
+topic_3: /edgetpu_human_pose_estimator/output/image
+timeout_3: 10
+
+topic_4: /edgetpu_object_detector/output/image
+timeout_4: 10
+
+topic_5: /gripper/imu
+timeout_5: 10
+
+topic_6: /head_camera/depth/image_raw
+timeout_6: 10
```

Comment:
how about adding rgb image, too?

Reply:
Thank you for your review!
I add `/head_camera/rgb/image_raw`